tools/perf/benchmark.common.js

```diff
@@ -25,7 +14,7 @@ function runTest(env, name, args, fncs, results) {
       throw Error("Junk in syntax parser result!");
     }
 
-    results[`${testName}/"${name}"`] = env.ms(end) - env.ms(start);
+    results[`${testName}/"${env.sampleName}"`] = env.ms(end) - env.ms(start);
```

Comment:
Since we're touching this code again, can we remove the quotes around the benchmark name?